static/src/stylesheets/layout/new-header/_veggie-burger.scss

```diff
@@ -58,6 +58,25 @@
     }
 }
 
+.veggie-burger__label {
+    @include fs-textSans(5);
+    bottom: -24px;
+    color: $neutral-3;
+    font-size: 16px;
```

Comment:
I spoke too soon! (or approved too soon)

It looks like you don't actually want to use the font mixin, but maybe just use a font variable instead